scripts/read_cts_output.py

```diff
@@ -0,0 +1,34 @@
+"""
+ Copyright (C) 2023 Intel Corporation
+
+ Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+ See LICENSE.TXT
+ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+"""
+# Print conformance test output from gtest, which is captured by the cts_exe.py script.
+
+import os
+import sys
+
+def print_file_content(folder_path, conformance_folder):
```

Comment:
`conformance_folder` is the last part of the `folder_path`. You can get it like this I suppose:
```
os.path.basename(folder_path)
```
Then there would be no need to pass it as the argument to this function.

Reply:
outdated